contents/data-display/timeline/middle-line/metadata.json

```
@@ -0,0 +1,17 @@
+{
+  "en": {
+    "title": "With line in the middle",
+    "description": "This is a timeline component that has a line in the middle."
+  },
+  "ja": {
+    "title": "中央に線があるタイムライン",
+    "description": "中央に線があるタイムラインコンポーネントです。"
+  },
+  "options": {
+    "container": {
+      "p": "md",
+      "centerContent": true
+    }
```

I think `calendar-icon` is the lowest priority.

```suggestion
    },
    "files": {
      "order": [
        "index.tsx",
        "timeline-item.tsx",
        "timeline.ts",
        "calendar-icon.tsx"
      ]
```

---

Fixed.